test/integration/test_mpsa_partial.py

```diff
@@ -4,144 +4,6 @@
 import porepy as pp
 
 
-class TestUpdateDisc(unittest.TestCase):
```

Comment:
What happened to the test?

Reply:
Good catch. It must have been deleted by an accident

Reply:
Never mind, I now remember. It was deleted on purpose. This test does exactly the same as the other class in the same file. There was a purpose of two tests for the early versions of displacement reconstruction, but at the current version there is no difference between the two tests.